frontend/src/components/events/PostCard.tsx

```diff
@@ -25,6 +36,7 @@ export function Card({ event }: { event: Event }) {
           {event.start_time ? timeFromNow(event.start_time) : ''}
         </span>
       </div>
+      <button onClick={handleRVSPButton}>OPEN</button>
```

Comment:
are we styling this rn?

Reply:
i just added some simple styling

![Screenshot 2024-07-08 052236](https://github.com/xusandrew/cs-348-project/assets/109555912/8c524586-317c-44ad-971e-6906614a7a9f)